index.html

```diff
@@ -16,10 +16,22 @@
     <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-    <header class="container">
-        <div class="row">
-        HEADER
-        </div>
+    <header class="header-fixed">
+        <nav class="navbar">
+            <div class="row">
+                <div class="logo-text">
```

Comment:
Nesu įsitikinęs, ar šis ```div``` yra reikalingas, galima būti išsiversti tik su viduje esančiu ```a``` elementu.